### PR TITLE
Feature: Trainee Page Dark Mode

### PR DESCRIPTION
**Pull request by arthurjunior250**
**What does this PR do?**  
-A user will be able to switch from dark to light theme
Light Theme
Screenshot_2022-11-15_09-41-22
Dark Theme
Screenshot_2022-11-15_10-55-23

### CODE REVIEW

File: src/hooks/useDarkMode.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
```
Comment (capitale250) [inline src/hooks/useDarkMode.ts]: why have we changed a hook??

File: src/components/sidebar/navHeader.tsx [hunk 2]
```diff
@@ -1,45 +1,51 @@
...
 const profile: string = require("../../assets/avatar.png").default;
-
-function NavBar() {
+const LogoWhite: string = require("../../assets/logoWhite.svg").default;
+import useDarkMode from '../../hooks/useDarkMode';
+const NavBar= forwardRef(({ ...props }: any, ref: any) => {
   const [showNotification, setShowNotification] = useState(false);
   const [showProfileDropdown, setShowprofileDropdown] = useState(false);
-  const { theme, setTheme } = useTheme()
-   function handleToggleTheme() {
```
Comment (capitale250) [inline src/components/sidebar/navHeader.tsx]: why have we changed the functionality

**Review by capitale250 — CHANGES_REQUESTED** [review]

File: src/hooks/useDarkMode.ts (hunk 1 above)
Comment (istafrica-dennw) [reply src/hooks/useDarkMode.ts]: @arthurjunior250 , i also see on the dark mode, the traine record dissapear

File: src/components/sidebar/navHeader.tsx (hunk 2 above)
Comment (arthurjunior250) [reply src/components/sidebar/navHeader.tsx]: corrected

Comment (arthurjunior250) [reply src/components/sidebar/navHeader.tsx]: corrected

**Pull request merged**